<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSource_Select">
    <property name="title">
     <string>Source Select</string>
    </property>
    <addaction name="actionFiles_from_disk"/>
    <addaction name="actionFolder_from_disk"/>
    <addaction name="actionCamera"/>
    <addaction name="actionCamera_Stream"/>
    <addaction name="actionSave_Camera_Stream"/>
   </widget>
   <widget class="QMenu" name="menuLabel_Images">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLabel_Images"/>
    <addaction name="actionLink_Image_Views"/>
    <addaction name="actionUse_colormap"/>
    <addaction name="actionMerge_image_displays"/>
    <addaction name="actionRegistration"/>
    <addaction name="actionDraw_menu"/>
   </widget>
   <widget class="QMenu" name="menuBatch">
    <property name="title">
     <string>Batch</string>
    </property>
    <addaction name="actionApply_filters_to_all_iamges"/>
    <addaction name="actionExtract_Stats_for_all_images"/>
   </widget>
   <addaction name="menuSource_Select"/>
   <addaction name="menuLabel_Images"/>
   <addaction name="menuBatch"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFiles_from_disk">
   <property name="text">
    <string>Files from disk</string>
   </property>
  </action>
  <action name="actionFolder_from_disk">
   <property name="text">
    <string>Folder from disk</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Snap camera</string>
   </property>
  </action>
  <action name="actionCamera_Stream">
   <property name="text">
    <string>Camera Stream</string>
   </property>
  </action>
  <action name="actionToggle_Source_View">
   <property name="text">
    <string>Toggle Source View</string>
   </property>
  </action>
  <action name="actionFilter_Images">
   <property name="text">
    <string>Filter Images</string>
   </property>
  </action>
  <action name="actionLabel_Images">
   <property name="text">
    <string>Label Images</string>
   </property>
  </action>
  <action name="actionApply_filters_to_all_iamges">
   <property name="text">
    <string>Apply filters to all images</string>
   </property>
  </action>
  <action name="actionExtract_Stats_for_all_images">
   <property name="text">
    <string>Extract Stats for all images</string>
   </property>
  </action>
  <action name="actionSave_Camera_Stream">
   <property name="text">
    <string>Save Camera Stream</string>
   </property>
  </action>
  <action name="actionLink_Image_Views">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Link Image Views</string>
   </property>
  </action>
  <action name="actionUse_colormap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use colormap</string>
   </property>
  </action>
  <action name="actionMerge_image_displays">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Merge image displays</string>
   </property>
  </action>
  <action name="actionRegistration">
   <property name="text">
    <string>Registration</string>
   </property>
  </action>
  <action name="actionDraw_menu">
   <property name="text">
    <string>Draw menu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
